internal/sdkprovider/service/kafkatopic/kafka_topic.go

```diff
@@ -275,6 +278,25 @@ func ResourceKafkaTopic() *schema.Resource {
 				}
 			}
 
+			// Validates topic conflict for new topics
+			if d.Id() != "" {
```

Comment:
why not `d.IsNewResource()` instead?

Reply:
`schema.ResourceDiff` doesn't have such method.